CHANGELOG.md

```diff
@@ -1,3 +1,35 @@
+## 0.10.0 / 2017-11-09
+* [ENHANCEMENT] Improve amtool check-config use and description text (#1016)
+* [ENHANCEMENT] Add metrics about current silences and alerts (#998)
+* [ENHANCEMENT] Sorted silences based on current status (#1015)
+* [ENHANCEMENT] add metric of alertmanager position in mesh (#1024)
+* [ENHANCEMENT] Initialise notifications_total and notifications_failed_total (#1011)
+* [ENHANCEMENT] Allow selectable matchers on silence view (#1030)
+* [ENHANCEMENT] Rename Promtheus to Alertmanager in CONTRIBUTING.md (#1033)
```

Comment:
Remove, not interesting for users.

Reply:
👍 i started dropping things halfway through, should have cleaned up the previous ones